modules/react-native-status/android/src/main/java/im/status/ethereum/module/AccountManager.java

```diff
@@ -0,0 +1,402 @@
+
+        try {
+            final File newFile = new File(dataFolder);
+            // todo handle error?
+            newFile.mkdir();
+        } catch (Exception e) {
+            Log.e(TAG, "error making folder: " + dataFolder, e);
+        }
+
+        final String ropstenFlagPath = this.utils.pathCombine(absRootDirPath, "ropsten_flag");
+        final File ropstenFlag = new File(ropstenFlagPath);
+        if (!ropstenFlag.exists()) {
+            try {
+                final String chaindDataFolderPath = this.utils.pathCombine(dataFolder, "StatusIM/lightchaindata");
+                final File lightChainFolder = new File(chaindDataFolderPath);
+                if (lightChainFolder.isDirectory()) {
+                    String[] children = lightChainFolder.list();
+                    for (int i = 0; i < children.length; i++) {
+                        new File(lightChainFolder, children[i]).delete();
+                    }
+                }
+                lightChainFolder.delete();
+                ropstenFlag.createNewFile();
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        }
+
+        String testnetDataDir = dataFolder;
+        String oldKeystoreDir = this.utils.pathCombine(testnetDataDir, "keystore");
+        String newKeystoreDir = this.utils.pathCombine(absRootDirPath, "keystore");
+        final File oldKeystore = new File(oldKeystoreDir);
+        if (oldKeystore.exists()) {
+            try {
+                final File newKeystore = new File(newKeystoreDir);
+                copyDirectory(oldKeystore, newKeystore);
+
+                if (oldKeystore.isDirectory()) {
+                    String[] children = oldKeystore.list();
+                    for (int i = 0; i < children.length; i++) {
+                        new File(oldKeystoreDir, children[i]).delete();
+                    }
+                }
+                oldKeystore.delete();
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        }
+
+        try {
+            final String multiaccountKeystoreDir = this.utils.pathCombine("/keystore", keyUID);
+            final String updatedJsonConfigString = this.updateConfig(jsonConfigString, absRootDirPath, multiaccountKeystoreDir);
+
+            prettyPrintConfig(updatedJsonConfigString);
+
+            return updatedJsonConfigString;
+        } catch (JSONException e) {
+            Log.e(TAG, "updateConfig failed: " + e.getMessage());
+            System.exit(1);
+
+            return "";
+        }
+    }
+
+    @ReactMethod
+    public void prepareDirAndUpdateConfig(final String keyUID, final String config, final Callback callback) {
+        Log.d(TAG, "prepareDirAndUpdateConfig");
+        String finalConfig = prepareDirAndUpdateConfig(config, keyUID);
+        callback.invoke(finalConfig);
+    }
+
+    //TODO : maybe nuke since it is not called anywhere in status-mobile code
```

Comment:
I missed nuking these, will take care of these

Reply:
cleaned up here : https://github.com/status-im/status-mobile/pull/18385/commits/1cd73b0242def75538da0e0b3eee2ee18304e6ca